package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -82,9 +82,9 @@ detect_mac80211() {
 
 		iw phy "$dev" info | grep -q 'Capabilities:' && htmode=HT20
 
-		iw phy "$dev" info | grep -q '5180 MHz' && {
+		iw phy "$dev" info | grep -q '* 5... MHz \[' && {
```

Comment:
I wonder whether the asterisk needs to be escaped here.

Reply:
I think you are right.
![image](https://user-images.githubusercontent.com/2470733/93509221-b8537100-f91f-11ea-8122-33e9fda96f28.png)

I updated the patch.